hplsql/src/main/java/org/apache/hive/hplsql/functions/FunctionMisc.java

```diff
@@ -211,6 +269,16 @@ public void partCount(HplsqlParser.Expr_spec_funcContext ctx) {
     query.close();
   }
 
+  public void modulo(HplsqlParser.Expr_func_paramsContext ctx) {
+    if (ctx.func_param().size() == 2) {
+      int a = evalPop(ctx.func_param(0).expr()).intValue();
+      int b = evalPop(ctx.func_param(1).expr()).intValue();
+      evalInt(a % b);
+    } else {
+      evalNull();
```

Comment:
Should exception be thrown?

Reply:
These are the APIs removed as a part of [HIVE-27492](https://issues.apache.org/jira/browse/HIVE-27492) fix. So just added back them again.Didn't do any code changes.